src/entity/Quiz.java

```diff
@@ -5,18 +5,26 @@
 import org.bson.codecs.pojo.annotations.BsonProperty;
 import org.bson.types.ObjectId;
 
+import java.time.LocalDateTime;
 import java.util.List;
 
 public class Quiz {
     @BsonId
     private ObjectId id;
+
     private String title;
     List<Question> questions;
+
+    @BsonProperty("creationTime")
+    private final LocalDateTime creationTime;
+
+    @BsonProperty("numberOfQuestions")
```

Comment:
why numberOfQuestions and quizLength instead of a consistent name (either sounds fine to me) for this property?

Reply:
Wait yeah good catch, I'll change it. Since quizLength is used more often in the other parts of the Quiz entity, I'll change it to quizLength.